pkg/types/report.go

```diff
@@ -74,6 +74,15 @@ const (
 	FormatCosignVuln Format = "cosign-vuln"
 )
 
+var BuildInK8sCompiances = []string{
```

Comment:
nit
```suggestion
var BuiltInK8sCompiances = []string{
```

Reply:
Thanks! fixed in https://github.com/aquasecurity/trivy/pull/8881/commits/94175bd699dcaad7cc537ca97f6f59d92681fd2a